src/components/fileviewer.js

```diff
@@ -145,15 +145,15 @@ export default class FileViewerContainer extends Component {
   }
 
   render() {
-    const { status, parsedFile, coverage, selectedLine } = this.state;
+    const { parsedFile, coverage, selectedLine } = this.state;
 
     return (
       <div>
         <div className="file-view">
           <FileViewerMeta
             revision={this.revision}
             path={this.path}
-            status={this.status}
+            self={this}
```

Comment:
`render()` is only called once, like a constructor.  Anything passed to it must exist before `componentWillMount()`, which means there is no `this.status` to pass yet.  We must pass the whole component and deference `status` later.